tests/test_opSimOrdering.py

```diff
@@ -0,0 +1,76 @@
+import os
+import numpy as np
+import pandas as pd
+from desc.twinkles import OpSimOrdering
+from lsst.utils import getPackageDir
+import unittest
+
+class TestOpSimOrdering(unittest.TestCase):
+
+    def setUp(self):
+        self.opSimDBPath = '/Users/rbiswas/data/LSST/OpSimData/minion_1016_sqlite.db'
```

Comment:
Yes, I think this can be done by reverting to the setup file that I had created in the generate PhoSim Input script pull request..